pkg/expr/sql_command.go

```diff
@@ -111,11 +119,26 @@ func (gr *SQLCommand) Execute(ctx context.Context, now time.Time, vars mathexp.V
 		return rsp, nil
 	}
 
-	rsp.Values = mathexp.Values{
-		mathexp.TableData{Frame: frame},
-	}
+	switch gr.format {
+	case "alerting":
```

Comment:
this currently can't be set via the UI, but you can use panel edit, or edit the queries in a saved alert rule.